src/ao_dens/rsp_general.f90

```diff
@@ -684,12 +688,14 @@ subroutine openrsp_get_property(n_props, np, pert_dims, pert_first_comp, &
        ! NOTE: This routine is placed here due to access to index/addressing routines
        ! Should be moved to API level once index/addressing routines are abstracted
        
-       open(unit=260, file='rsp_tensor', status='replace', action='write') 
+       funit = get_safe_funit()
+       
+       open(unit=funit, file='rsp_tensor', status='replace', action='write') 
```

Comment:
The following is a better way to open a new unit:
```
       open(newunit=funit, file='rsp_tensor', status='replace', action='write')
```

Reply:
I don't understand this

Reply:
the difference is that I use some attribute "newunit" instead of "unit"?

Reply:
ok, so I went on the interactive world wide web and I found it out myself, who could have guessed healthcare was so complicated

Reply:
It is a "new" feature since Fortran 2008 standard. Using `newunit` instead of `unit` will return an automatically chosen, unused unit number.

Reply:
it appears to be a Fortran 2008 feature, are we OK with requiring that?

Reply:
I would say it is safe. AFAIK all major compilers have supported it for a quite a while now. It's 12 years old so one should hope so. GCC has had it since 4.9, which AFAIK is not even supported by neither Dalton nor LSDalton.

Reply:
You could check whether or not you're not already requiring Fortran 2008 by compiling with gfortran and using the "-std=f2008"  compiler option.